dataprofiler/labelers/classification_report_utils.py

```diff
@@ -342,15 +343,15 @@ def classification_report(
     p, r, f1, s = precision_recall_fscore_support(
         MCM, labels=labels, average=None, sample_weight=sample_weight
     )
-    rows = zip(target_names, p, r, f1, s)
+    rows = zip(target_names, p, r, f1, cast(np.ndarray, s))
 
     if y_type.startswith("multilabel"):
-        average_options = ("micro", "macro", "weighted", "samples")
+        average_options: Tuple[str, ...] = ("micro", "macro", "weighted", "samples")
```

Comment:
why the `...` do we not want to enforce a len of four? so something like `Tuple[str, str, str, str]`?

Reply:
The next line is a tuple with length 3

Reply:
https://docs.python.org/3/library/typing.html#typing.Tuple

Reply:
```To specify a variable-length tuple of homogeneous type, use literal ellipsis, e.g. Tuple[int, ...]. A plain [Tuple](https://docs.python.org/3/library/typing.html#typing.Tuple) is equivalent to Tuple[Any, ...], and in turn to [tuple](https://docs.python.org/3/library/stdtypes.html#tuple).```